invenio_rdm_records/services/schemas/metadata.py

```diff
@@ -217,16 +217,43 @@ def _is_uri(uri):
         return False
 
 
+class URLSchema(Schema):
+    """Schema for the URL schema."""
+
+    url = SanitizedUnicode(
+        validate=_valid_url(_('Not a valid URL.'))
+    )
+
+
 class RightsSchema(Schema):
     """License schema."""
 
     id = SanitizedUnicode()
-    title = SanitizedUnicode()
-    description = SanitizedUnicode()
+    title = fields.Dict()
+    description = fields.Dict()
+    props = fields.Nested(URLSchema)
     link = SanitizedUnicode(
         validate=_valid_url(_('Not a valid URL.'))
     )
 
+    @validates_schema
+    def validate_rights(self, data, **kwargs):
+        """Validates that id xor name are present."""
+        id_ = data.get("id")
+        title = data.get("title")
+
+        if not id_ and not title:
+            raise ValidationError(
+                _("An existing id or a free text title must be present"),
+                "rights"
+            )
+        elif id_ and len(data.values()) > 1:
```

Comment:
I tested in the form of the demo instance and rights allows multiple to be added. Is this still correct?

![Screenshot 2021-07-15 at 11 58 08](https://user-images.githubusercontent.com/6756943/125769361-c6d1983c-70b3-488b-800e-1288c2861579.png)

Reply:
Yes, that's correct

Reply:
Yes, it checks that for vocab values we only have `id` present